### PR TITLE
uuidd: add page

### PR DESCRIPTION
**Pull request by SethFalco**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Relates to: https://github.com/tldr-pages/tldr/issues/2214

### CODE REVIEW

File: pages/linux/uuidd.md
Comment (CleanMachine1) [inline pages/linux/uuidd.md:16]: Does there not need to be a time parameter

File: pages/linux/uuidd.md
Comment (CleanMachine1) [inline pages/linux/uuidd.md:10]:
```suggestion
- Generate a bulk number of random UUIDs:
```

**Review by CleanMachine1 — APPROVED** [review]
Has my approval after my 2 reviews are looked at

**Comment by CleanMachine1** [comment]
You are probably welcome to become a maintainer of tldr-pages, since this is your 7th to be approved PR

**Comment by bl-ue** [comment]
Yeah, @SethFalco if you'd be interested in helping out with reviewing/approving PRs, I'd be happy to open an issue inviting you!

If you'd rather not right now, by all means no worries 💯 :)

File: pages/linux/uuidd.md
Comment (bl-ue) [reply pages/linux/uuidd.md:16]: Not sure what you're getting at, @CleanMachine1.

Comment (SethFalco) [reply pages/linux/uuidd.md:16]: I can vouch it's not required at least, I've tried to find more information on this but didn't see anything, and the man pages make no mention of a value for the argument.

```log
seth@Rogeliina:~$ uuidd --time
7a03154c-b82b-11eb-8d10-8df892758bb2
seth@Rogeliina:~$ uuidd --time
7b1946fe-b82b-11eb-8d10-8df892758bb2
seth@Rogeliina:~$ uuidd --time
7ba80326-b82b-11eb-8d10-8df892758bb2
```

When testing it with a real values, the behavior doesn't change.

I've tried to peek again before making this comment, but can only assume based on information on UUID v1 that it generates one using the current timestamp, and that's all it supports.

> Version 1 concatenates the 48-bit MAC address of the "node" (that is, the computer generating the UUID), with a 60-bit timestamp, being the number of 100-nanosecond intervals since midnight 15 October 1582 Coordinated Universal Time (UTC), the date on which the Gregorian calendar was first adopted.
> \- https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_1_(date-time_and_MAC_address)

I could update this to specify that it uses the current timestamp and MAC address, though.

```diff
- - Generate a time based UUID:
+ - Generate a time based UUID, based on the time and MAC address of the system:
```

**Comment by SethFalco** [comment]
> Yeah, @SethFalco if you'd be interested in helping out with reviewing/approving PRs, I'd be happy to open an issue inviting you!

If everyone else is cool with it, I've love to help maintain the repository. Big fan of tldr! \\(^-^)/

Just a forewarning through in case it's a concern. I'm a volunteer of some other projects, and picking up a full-time job soon, is it alright if there are pockets of inactivity?

**Comment by CleanMachine1** [comment]
Of course!

File: pages/linux/uuidd.md
Comment (CleanMachine1) [reply pages/linux/uuidd.md:16]: To be honest I don't even know what uuid is and how to use it on linux

Need to look at the tldr page 😁

**Comment by CleanMachine1** [comment]
If you do become a collaborator, are you fine with me pinging you for reviews and opinions, I am sure this applies for others as well

**Comment by SethFalco** [comment]
> If you do become a collaborator, are you fine with me pinging you for reviews and opinions, I am sure this applies for others as well

Absolutely! For on-demand stuff or where I'm explicitly mentioned, you can consider me available 24/7.
The "pockets of inactivity" was only aimed for future, since I suspect I'll get preoccupied with projects or tasks sometimes.

**Comment by bl-ue** [comment]
@SethFalco pockets of inactivity are more than allowed — a volunteer does everything for free and under no obligation!

That would be super, I'll go open an issue... 🎉

>picking up a full-time job soon

Cool—I wish you good luck! :) Remote, right?

File: pages/linux/uuidd.md
Comment (bl-ue) [inline pages/linux/uuidd.md:14]:
```suggestion
- Generate a time-based UUID, based on the current time and MAC address of the system:
```

**Review by bl-ue — APPROVED** [review]

**Pull request merged**
